doc/user/content/sql/create-connection.md

```diff
@@ -609,6 +609,84 @@ CREATE CONNECTION csr_ssh TO CONFLUENT SCHEMA REGISTRY (
 {{< /tab >}}
 {{< /tabs >}}
 
+### MySQL
+
+{{< private-preview />}}
+
+A MySQL connection establishes a link to a [MySQL] server. You can use
+MySQL connections to create [sources](/sql/create-source/mysql).
+
+#### Syntax {#mysql-syntax}
+
+{{< diagram "create-connection-mysql.svg" >}}
+
+#### Connection options {#mysql-options}
+
+Field                       | Value            | Required | Description
+----------------------------|------------------|:--------:|-----------------------------
+`HOST`                      | `text`           | ✓        | Database hostname.
+`PORT`                      | `integer`        |          | Default: `3306`. Port number to connect to at the server host.
+`USER`                      | `text`           | ✓        | Database username.
+`PASSWORD`                  | secret           |          | Password for the connection.
+`SSL CERTIFICATE AUTHORITY` | secret or `text` |          | The certificate authority (CA) certificate in PEM format. Used for both SSL client and server authentication. If unspecified, uses the system's default CA certificates.
+`SSL MODE`                  | `text`           |          | Default: `disabled`. Enables SSL connections if set to `required`, `verify_ca`, or `verify_identity`. See the [MySQL documentatation](https://dev.mysql.com/doc/refman/8.0/en/using-encrypted-connections.html) for more details.
```

Comment:
Probably also for `preferred`?

Reply:
`Preferred` is explicitly not supported because of security concerns ([source code](https://github.com/MaterializeInc/materialize/blob/d5a642f52ecdede3e363dfe5521fe401c05abc22/src/sql/src/plan/statement/ddl/connection.rs#L409)).